.github/workflows/main.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@v1
       with:
-        python-version: 3.8
+        python-version: 3.9
```

Comment:
At some point, we should use the GitHub actions matrix to parameterise this and test more than a single Python version.